jujugui/static/gui/src/app/models/entity-extension.js

```diff
@@ -18,19 +18,20 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 'use strict';
 
-YUI.add('search-result-extension', function(Y) {
+YUI.add('entity-extension', function(Y) {
```

Comment:
Realized that this extension is more generically re-usable beyond just search results. We need pretty much the same POJO display object in the details page. Renamed it to `entity-extension` and clarified that it holds functionality (not just producing the display object) common to both charms and bundles.